src/components/LifeEventsListView.vue

```diff
@@ -118,6 +158,26 @@ export default {
             if (this.includeUpcomingEvents && !isFuture(e.redZone.start)) return true
             if (! this.includeUpcomingEvents && !isFuture(e.redZone.start)) return true
           })
+          .filter(e => {
+            if(this.filterTag === undefined) return true
+            console.log(this.filterTag)
+            if(e.title.includes("#"+this.filterTag)) return true
+            return false
+          })
+    },
+    listOfCurrentTags() {
+      let tags = []
+      this.events.forEach(element => {
```

Comment:
Die verfügbaren/clickbaren Tags sind ja momentan abhängig von den Event die aktuell in der Event-List dargestellt werden. Du hast das im Call als Feature definiert.

Evt. willst du die UX mal testen mit der gesamten Event-Liste als Basis? Dann bliebe die Tag-Liste immer stabil. Dazu könntest du hier einfach die momentane Basis `this.events` ersetzen mit `this.sortedEvents(this.sortBy)`.

Ist am Ende deine Entscheidung da nicht spezifiziert :D

Reply:
Ich habs probiert, dann bekommst du halt auch tags angezeigt, deren Todos nicht sichtbar sind (nicht in Red Zone). Also ich habs jetzt mal auf die gefilterte Liste belassen.

Reply:
Guter Punkt.

Eine denkbare Lösung wäre dann mit 2 Event-Listen zu arbeiten:

1. `this.events` so belassen wie vor deinem PR. Also nur die Filter mit den switches. Als Name vielleicht `eventContext`
2. Auf diesem `Context` kann dann dein neuer Tag-Filter angewandt werden. Diese 2. computed property würde dann oben im Template referenziert werden als Liste der darzustellenden Events..

Reply:
Vergiss den obigen Vorschlag. Leichter und verständlicher wäre wohl:

1. Tag Suche startet auf Basis alles Events
2. Dann werden alle geschlossen Event weg gefiltert

Die verbleibende Menge ist relevant für Tags da 
a) offene Events immer sichtbar sind, man kann sie aktuell nicht wegfilter (afaik)
b) mir diese Menge interessiert den User für seine Tags